.NET/Microsoft.Recognizers.Text.DateTime/Extractors/IDateTimeExtractor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Enable a reference time pass to the extractor
+
+using DateObject = System.DateTime;
+using System.Collections.Generic;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    public interface IDateTimeExtractor : IExtractor
```

Comment:
Can't this interface be an abstract class and include the default implementation of Extract?